src/features/confirm/Confirm.js

```diff
@@ -68,23 +73,15 @@ export const ConfirmContainer = () => {
       .unwrap()
       .then(res => {
         if (isSucessful(res)) {
-          clearInterval(queryPaymentStatusInterval);
-          navigate(`/status/success?reason=${name}`, { replace: true });
+          clearAndNavigate(`/status/success?reason=${name}`);
           return;
         }
-        if (isProcessing(res)) {
-          if (queryPaymentStatusTry < 3) {
-            queryPaymentStatusTry++;
-          } else {
-            clearInterval(queryPaymentStatusInterval);
-            const timoutMessage = 'Payment status query is over 3 times';
-            navigate(`/status/error?reason=${timoutMessage}`, { replace: true });
-          }
+        if (isProcessing(res) && queryPaymentStatusTry < 3) {
+          queryPaymentStatusTry++;
           return;
         }
-        clearInterval(queryPaymentStatusInterval);
-        const errorMessage = res.sub_return_message || res.return_message;
-        navigate(`/status/error?reason=${errorMessage}`, { replace: true });
+        const errorMessage = isProcessing(res) ? 'Payment status query is over 3 times' : res.sub_return_message || res.return_message;
```

Comment:
@longxice any better way for the message: `Payment status query is over 3 times` ?